src/meta/src/hummock/manager/mod.rs

```diff
@@ -897,6 +891,51 @@ where
         Ok(())
     }
 
+    fn is_compact_task_expired(compact_task: &CompactTask, levels: &Levels) -> bool {
```

Comment:
Better to add some comments to clarify the conditions to determine whether a task is expired.

Reply:
> Better to add some comments to clarify the conditions to decide whether a task is expired.

just refactored. Now this part of code is more clear. :-)